### PR TITLE
[Model] Refactor Qwen2-VL to use merged multimodal processor

### PR DESCRIPTION
**Pull request by Isotr0py**

### CODE REVIEW

File: vllm/model_executor/models/qwen2_vl.py [hunk 1]
```diff
@@ -799,290 +737,88 @@ def get_max_qwen2_vl_mm_tokens(ctx: InputContext,
...
+        self,
+        *,
+        min_pixels: Optional[int] = None,
+        max_pixels: Optional[int] = None,
+    ) -> ProcessorMixin:
+        hf_processor = self.ctx.get_hf_processor()
+        if min_pixels:
+            hf_processor.image_processor.min_pixels = min_pixels
+        if max_pixels:
+            hf_processor.image_processor.min_pixels = max_pixels
```
Comment (DarkLight1337) [inline vllm/model_executor/models/qwen2_vl.py:752]:
```suggestion
            hf_processor.image_processor.max_pixels = max_pixels
```

File: vllm/model_executor/models/qwen2_vl.py [hunk 2]
```diff
@@ -742,54 +671,31 @@ def _get_vision_info(
...
-        data_type_key=data_type_key,
-        mm_count=mm_count,
-    )
-
-
 def get_max_qwen2_vl_mm_tokens(ctx: InputContext,
                                data_type_key: str,
                                *,
                                min_pixels=None,
                                max_pixels=None) -> int:
```
Comment (DarkLight1337) [inline vllm/model_executor/models/qwen2_vl.py:688]:
```suggestion
                               min_pixels: Optional[int] = None,
                               max_pixels: Optional[int] = None) -> int:
```

Technically not part of this PR, but might as well fix it.

**Review by DarkLight1337 — APPROVED** [review]
LGTM, thanks for the refactor!

**Comment by DarkLight1337** [comment]
The failure in embeddings input test should be fixed by the Qwen2-Audio PR.

**Comment by B-201** [comment]
Hi, I encountered some issues while using this PR:
image

Is this caused by the version of hf? I'm using 4.46.0.

**Comment by DarkLight1337** [comment]
> Hi, I encountered some issues while using this PR:
> image
> 
> Is this caused by the version of hf? I'm using 4.46.0.

I think so; try updating to v4.47.

**Comment by B-201** [comment]
> > Hi, I encountered some issues while using this PR:
> > image
> > Is this caused by the version of hf? I'm using 4.46.0.
> 
> I think so; try updating to v4.47.

It works, thank you!
